frontend/src/component/common/util.ts

```diff
@@ -113,14 +114,18 @@ export function updateWeightEdit(
         { remainingPercentage: totalWeight, variableVariantCount: 0 }
     );
 
-    const percentage = parseInt(
-        String(remainingPercentage / variableVariantCount)
-    );
+    const getPercentage = () =>
+        Math.round(remainingPercentage / variableVariantCount);
```

Comment:
When `variableVariantCount == 1` it will return `remainingPercentage`, so there is no special case at the end